client/signup/utils.js

```diff
@@ -3,6 +3,7 @@
  */
 import { filter, find, includes, isEmpty, pick, sortBy } from 'lodash';
 import { translate } from 'i18n-calypso';
+import { createExPlatClient } from '@automattic/explat-client';
```

Comment:
@StefanNieuwenhuis, is there a reason for choosing this approach over the one used by @jessie-ross during the AA test in https://github.com/Automattic/wp-calypso/pull/50618?

That seems pretty straight-forward to me but I may be missing something:
- [Loading experiment](https://github.com/Automattic/wp-calypso/pull/50618/files#diff-8460f0344f9204b42fc128f16c269b888163a7310914a4dacda3995cefda6380)
- [Checking for variation](https://github.com/Automattic/wp-calypso/pull/50618/files#diff-9d7d6e5b810ec4d930d0dd4ab43ab0a72e8082274e818ed6888ec938a94dacc2)

Reply:
Maybe I need to fix some documentation somewhere? The readme in `@automattic/explat-client`?

Reply:
> Maybe I need to fix some documentation somewhere? The readme in `@automattic/explat-client`?

The _In React / Outside React_ brief summary from pbmo2S-HC-p2 is really insightful and would probably help new folks that are looking at how to set an experiment anywhere in Calypso.